src/app/screens/createInscription/index.tsx

```diff
@@ -28,11 +28,100 @@ import type { UTXO } from '@secretkeylabs/xverse-core/types';
 import { getShortTruncatedAddress } from '@utils/helper';
 
 import useSeedVault from '@hooks/useSeedVault';
+import Callout from '@ui-library/callout';
+import { StyledP } from '@ui-library/common.styled';
 import CompleteScreen from './CompleteScreen';
 import ContentLabel from './ContentLabel';
 import EditFee from './EditFee';
 import ErrorModal from './ErrorModal';
 
+type CardRowProps = {
+  topMargin?: boolean;
+  center?: boolean;
+};
+const CardRow = styled.div<CardRowProps>((props) => ({
+  display: 'flex',
+  flexDirection: 'row',
+  alignItems: props.center ? 'center' : 'flex-start',
+  justifyContent: 'space-between',
+  marginTop: props.topMargin ? props.theme.spacing(8) : 0,
+}));
+
+const NumberWithSuffixContainer = styled.div((props) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  alignItems: 'flex-end',
+  color: props.theme.colors.white_0,
+}));
+
+const NumberSuffix = styled.div((props) => ({
+  ...props.theme.typography.body_s,
+  color: props.theme.colors.white_400,
+}));
+
+const StyledPillLabel = styled.p`
+  display: flex;
+  align-items: center;
+  gap: ${(props) => props.theme.space.s};
+`;
+
+const Pill = styled.span`
+  ${(props) => props.theme.typography.body_bold_s}
+  color: ${(props) => props.theme.colors.elevation0};
+  background-color: ${(props) => props.theme.colors.white_0};
+  padding: 3px 6px;
+  border-radius: 40px;
+`;
+
+function FeeRow({
+  label,
+  subLabel,
+  value = 0,
+  fiatCurrency,
+  fiatRate,
+  repeat,
+}: {
+  label: string;
+  subLabel?: string;
+  value?: number | string | null;
+  fiatCurrency: string;
+  fiatRate: string;
+  repeat?: number;
+}) {
+  if (!value) {
+    return null;
+  }
+  const fiatValue = new BigNumber(value || 0).dividedBy(100e6).multipliedBy(fiatRate).toFixed(2);
```

Comment:
(nit)

Can we add a comment here explaining the magic number `100e6` (sats to BRC I'm assuming)? Or better yet, make it a helper function?